test/index.js

```diff
@@ -663,6 +663,33 @@ describe('Bigtable', function() {
         done();
       });
 
+      it('should not call replace project ID token', function(done) {
+        replaceProjectIdTokenOverride = sinon.spy();
+
+        bigtable.api[CONFIG.client][CONFIG.method] = {
+          bind: function() {
+            assert(!replaceProjectIdTokenOverride.called);
+            setImmediate(done);
+
+            return common.util.noop;
+          },
+        };
+
+        bigtable.request(CONFIG, assert.ifError);
+      });
+    });
+
+    describe('replace projectID token', function() {
+      beforeEach(function() {
+        bigtable = new Bigtable();
+        bigtable.getProjectId_ = function(callback) {
+          callback(null, PROJECT_ID);
+        };
+
+        bigtable.api[CONFIG.client] = {
+          [CONFIG.method]: common.util.noop,
+        };
+      });
```

Comment:
Can you leave a new line to separate all `beforeEach`, `describe`, `it`, etc. blocks?

Reply:
@stephenplusplus, I fixed the new lines.  PTAL.